File: testprojects/master_scala_mainone/pipelines/SC_CONFIG_PIP/code/src/main/scala/io/prophecy/pipelines/sc_config_pip/graph/Reformat_1.scala
package io.prophecy.pipelines.sc_config_pip.graph

import io.prophecy.libs._
import io.prophecy.pipelines.sc_config_pip.config.ConfigStore._
import io.prophecy.pipelines.sc_config_pip.udfs.UDFs._
import io.prophecy.pipelines.sc_config_pip.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.select(
      concat(lit(Config.c_string), col("`c  - int`")).as("c1_string"),
      concat(col("`c  - int`"),    lit(Config.c_boolean)).as("c2_boolean"),
      concat(col("`c  - int`"),    lit(Config.c_double)).as("c3_double"),
      concat(col("`c  - int`"),    lit(Config.c_float)).as("c4_float"),
      concat(col("`c  - int`"),    lit(Config.c_short)).as("c5_short"),
      concat(col("`c  - int`"),    lit(Config.c_int)).as("c6_int"),
      expr(Config.c_spark_expression).as("c6_spark_expr")
    )

}

File: testprojects/streaming/python/pipelines/StreamingTestPipeline_1Clone/code/streamingtestpipeline_1/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingtestpipeline_1.config.ConfigStore import *
from streamingtestpipeline_1.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(
        col("c_tinyint"), 
        col("c_smallint"), 
        col("c_int"), 
        col("c_bigint"), 
        col("c_float"), 
        col("c_double"), 
        col("c_string"), 
        col("c_boolean"), 
        col("c_array"), 
        col("c_struct"), 
        col("p_int"), 
        col("p_float"), 
        col("p_string")
    )
